control-plane-operator/controllers/hostedcontrolplane/v2/assets/openshift-apiserver/deployment.yaml

```diff
@@ -56,12 +56,22 @@ spec:
         readinessProbe:
           failureThreshold: 10
           httpGet:
-            path: healthz
+            path: readyz?exclude=etcd&exclude=etcd-readiness
```

Comment:
We need to drop the `initialDelaySeconds` to `0` since there is a startup probe.  We should probably adjust the other probe values to match upstream.

Reply:
@Evan-Reilly ping in case you missed this.